### PR TITLE
Replace `--func-only` with `--warp-surfaces-native2std`

### PR DESCRIPTION
**Pull request by tsalo**
Closes None.

## Changes proposed in this pull request

- Invert the logic of `--func-only` and replace it with `--warp-surfaces-native2std`. Now, the surfaces won't be outputted by default, unless you're running `hcp` or `dcan` data.
- Split the steps that warp the t1w and t1seg files to MNI space out of the anatomical workflow and into a separate one (`init_t1w_wf`) that runs no matter what.
- Move the creation of the T1 brain sprite figure into a separate workflow as well (`init_brainsprite_wf`).
- So much refactoring!

### Breaking changes

- `space-MNI152NLin6Asym_desc-preproc_dseg.nii.gz` --> `space-MNI152NLin6Asym_dseg.nii.gz`
    - The dseg output no longer takes its name from the T1w file. It now uses its own original file as the name source. This means that fields unique to the T1w file (e.g., `desc-preproc` for fMRIPrep derivs) will go away.
- The anatomical workflow is no longer run by default.

### CODE REVIEW

File: xcp_d/utils/doc.py [hunk 1]
```diff
@@ -209,6 +209,34 @@
...
+        * - ``<source_entities>_space-fsLR_den-32k_hemi-<L|R>_hcpmidthickness.surf.gii``
+          - Something
+        * - ``<source_entities>_space-fsLR_den-32k_hemi-<L|R>_hcpveryinflated.surf.gii``
+          - Something
+        * - ``<source_entities>_space-fsLR_den-32k_hemi-<L|R>_midthickness.surf.gii``
+          - Something
+        * - ``<source_entities>_space-fsLR_den-32k_hemi-<L|R>_pial.surf.gii``
+          - Something
+        * - ``<source_entities>_space-fsLR_den-32k_hemi-<L|R>_smoothwm.surf.gii``
+          - Something
```
Comment (tsalo) [inline xcp_d/utils/doc.py:237]: @madisoth could you fill in the gaps on this? I don't know how to describe each, or what they're used for.

File: xcp_d/utils/doc.py [hunk 2]
```diff
@@ -209,6 +209,34 @@
     List of paths to atlas-specific ROI-to-ROI correlation files.
 """
 
+docdict["process_surfaces"] = """
+process_surfaces : :obj:`bool`, optional
+    If True, a workflow will be run to warp native-space (fsnative) reconstructed cortical
+    surfaces (surf.gii files) produced by Freesurfer into standard (fsLR) space.
+    These surface files are primarily used for visual quality control.
```
Comment (tsalo) [inline xcp_d/utils/doc.py:216]: @madisoth are they used for anything other than visual QC?

**Comment by tsalo** [comment]
@kahinimehta I've made the change you requested (selecting the brainsprite ribbon outside of the anatomical workflow), and tests are finally passing. When you have a chance, could you take another look?

**Review by kahinimehta — COMMENTED** [review]
Just confirming the right file is being used for the "ribbon", if the anat workflow is not run - as we spoke about yesterday

File: xcp_d/workflow/base.py [hunk 3]
```diff
@@ -97,7 +99,7 @@ def init_xcpd_wf(
                 work_dir=".",
                 dummytime=0,
                 fd_thresh=0.2,
-                func_only=False,
+                process_surfaces=True,
```
Comment (kahinimehta) [inline xcp_d/workflow/base.py:102]: isn't false the default?

**Review by kahinimehta — APPROVED** [review]

File: xcp_d/workflow/base.py (hunk 3 above)
Comment (tsalo) [reply xcp_d/workflow/base.py:102]: Good point! I'll switch it to the default.

**Pull request merged**